charts/redpanda/templates/secrets.yaml

```diff
@@ -80,9 +80,11 @@ stringData:
       touch /tmp/postStartHookFinished
     }
 
+  {{- if not ( dig "node" "recovery_mode_enabled" false .Values.config ) }}
     postStartHook
-    true
+  {{- end }}
 
+    true
```

Comment:
In my opinion this will backfire in the worst possible moment. Disable or exit from maintenance mode should be possible even in recovery mode.

Reply:
What does this mean:
```
backfire in the worst possible moment
```

Reply:
When you will be in the incident after long hours of the back and forth between multiple stakeholders then it will hard to remember to validate if maintenance mode was disabled (the main logic of postStartHook). In my opinion it should always run and Redpadna should accept that REST request.